pyproject.toml

```diff
@@ -10,6 +10,8 @@ Flask = "^2.2.5"
 python-dotenv = "^0.14.0"
 requests = "^2.31.0"
 pytest = "^8.1.1"
+pymongo = "^4.10.1"
+mongomock = "^4.2.0.post1"
```

Comment:
This odd looking version was new to me & I [learned something looking this up](https://peps.python.org/pep-0440/#post-releases)
> Some projects use post-releases to address minor errors in a final release that do not affect the distributed software (for example, correcting an error in the release notes).